### DISCUSSION TITLE
error while performing parameter identification (visual feedback)

### DISCUSSION THREAD

**Discussion by Fate-afshar** [Open-Systems-Pharmacology/Forum#1236] [Questions & Problems]
Hi. I'm a newbie so I apologize in advance if my question is basic 
after performing the parameter identification I don't seem to be getting an error but I see no charts or graphs.
uploaded a screenshot to further make my point.
help would be appreciated 
![Screenshot (147)](https://user-images.githubusercontent.com/108145488/175602069-3f258ad1-3cf6-4114-868f-e601bca1c4a9.png)

**Comment by msevestre** [comment]
There is a warning showing up. Can you go over with the mouse? What does it say?

**Reply by Fate-afshar** [reply]
there is no message or warning or error. hovering the mouse does nothing either

**Reply by msevestre** [reply]
yes there is one
![image](https://user-images.githubusercontent.com/1041237/175652189-6e78290b-0196-499a-ab76-714958c7cbd3.png)

What does it say? go over the icon on the row header . A message should be displayed. 
You can also attach your project so we can take a look

**Comment by Fate-afshar** [comment]
it says: too close to the boundary but I tried different parameter identifications that resulted in no warning but instead a green check. but still no chart or graph
I attached my project in a zip file
thanks again for helpin
[Vancomycin#3.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/8985183/Vancomycin.3.zip)
g

**Reply by msevestre** [reply]
Your PI just runs too fast and there is no time to actually refresh the view once. 
Changing the values to another start value allows for at least one view refresh
![image](https://user-images.githubusercontent.com/1041237/175784651-d21c136f-4830-4d8e-a3e5-501e21df3ac4.png)


It looks like your simulation is not sensitive at all to those parameters.

You can change the PI algorithm used to slow down the process a bit 
![image](https://user-images.githubusercontent.com/1041237/175784694-22416b30-88b9-4774-a634-ac4793d56c33.png)

That being said, if you are new to the software, starting right away with Parameter Identification is probably not the most effective way to understand your model. I would suggest going over some of the online materials, documentation etc... and make sure you understand what's going on before using a PI.

**Comment by Fate-afshar** [comment]
thanks a bunch! 
will do
